templates/WAM_APP_FRED/test_map_layout.html

```diff
@@ -472,8 +489,234 @@ <h2 class="tabs-title__head">Feedin-Time Series</h2>
+                    }
+                  })
+                  map.addLayer(layer)
+                }
+              });
+
+              //PowePlantRegister StartSection
+              ///////////////////////////////////////////////////////////////////////////////////////////////////////
+              // res PowerPlants in region boundaries
+
+              function enablePowPlPopup(feature, layer) {
+                layer.on('mouseover', function () {
+                  this.setStyle(highlightStyleMouseOver);
+                });
+                layer.on('click', function () {
+                  this.setStyle(highlightStyle);
+                });
+                layer.on('mouseout', function () {
+                  this.setStyle(wdataStyle);
+                });
+                  layer.bindPopup('Loading...');
+
+                  layer.on('click', onPowerPlantDataPointClick);
+              }
+              
+              var regionPowerPlantgroup = L.layerGroup().addTo(map);
+              // request all power plants conatined in selected region
+              // also include the generation_type option 
+              function loadRegionPowerPlants(region, layer){
+                region_name = region.feature.properties.name;
+                generation_type = $("input[name=generation_type]:checked").val();
+
+                $.ajax({
+                    url: ppr_location,
+                    type: 'POST',
+                    async: true,
+                    headers: {'X-CSRFToken': '{{ csrf_token }}'},
+                      data: {
+                      'region_name': region_name,
+                      'generation_type': generation_type
+                      },
+                    success: function(powerplant){
+                      var layer = new L.geoJSON(powerplant, {
+                        onEachFeature: enablePowPlPopup,
+                        pointToLayer: createCircles
+                      });
+                      //map.addLayer(layer)
+                      if (regionPowerPlantgroup.getLayers().length === 0) {
+                        regionPowerPlantgroup.addLayer(layer)
+                      } 
+                      else {
+                        regionPowerPlantgroup.clearLayers();
+                      }
+
+                    }
+                  });
+                }
+              
+              // add all power plants from url to map
+              function addPowerPlantToMap(){
+
+              }
+              
+              $.ajax({
+                    url: ppr_location,
+                    dataType: 'json',
+                    async: true,
+                    success: function(powerplant){
+
+                        var layer = new L.geoJSON(powerplant, {
+                            onEachFeature: enablePowPlPopup,
+                            pointToLayer: createCircles
+                        });
+                        map.addLayer(layer)
```

Comment:
why not adding the points in a separate layer and do it directly into map?

Reply:
yor are right sould add another layergroup, can fix this tomorrow morning